sdks/python/apache_beam/coders/standard_coders_test.py

```diff
@@ -74,7 +74,7 @@ class StandardCodersTest(unittest.TestCase):
           lambda x: IntervalWindow(
               start=Timestamp(micros=(x['end'] - x['span']) * 1000),
               end=Timestamp(micros=x['end'] * 1000)),
-      'urn:beam:coders:stream:0.1': lambda x, parser: map(parser, x),
```

Comment:
So this test is seems to indicate its for testing a stream but by putting list around it this isn't really a stream anymore? Or am I missreading the code here? (these tests are a little less than self-documenting maybe @vikkyrk who added the iterablecoder tests can chime in).